docs/proposals/dependencies-automatically-propagation/README.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Dependencies Automatically Propagation
+authors:
+- "@mrlihanbo"
+  reviewers:
+- "@TBD"
```

Comment:
Put @Garrybest @XiShanYongYe-Chang @RainbowMango @kevin-wangzefeng to the reviewer list
and put @RainbowMango @kevin-wangzefeng to the approver list.